cmd/arc_summary/arc_summary.py

```diff
@@ -705,8 +705,8 @@ def _l2arc_summary(Kstat):
         )
         sys.stdout.write("\n")
 
-        if arc["l2_arc_evicts"]['lock_retries'] + \
-                arc["l2_arc_evicts"]["reading"] > 0:
+        if arc["l2_arc_evicts"]['lock_retries'] != '0' or \
```

Comment:
Good question. I'm a fan of consistency and from that perspective it should be double quoted. If you however look at the 2 other mentions of lock_retries in this file they're all single quoted, so at the end of the day it depends on what your preference is.

Given this string, I'd be happy to flip this one to single quotes, or all of them. Looking through other strings in this file there's not a whole lot of consistency wrt single vs double quotes, so that could all be cleaned up. I'd be happy to take a stab at that if such a pull request would be valued.